### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by MichaelGNairn**

### CODE REVIEW

**Comment by ANikolova22** [comment]
Hey @MichaelGNairn , is this being reviewed by someone currently or it's still waiting to be picked up?

**Comment by MichaelGNairn** [comment]
@ANikolova22 not a clue!

**Comment by ANikolova22** [comment]
Ok, just checking as I didn't find a card for it. I can pick it up

**Comment by ANikolova22** [comment]
Hey @MichaelGNairn - given that this is just a minor update to labels etc., I only ran the test version with example input. Some issues encountered:
- Library errors: given that I have a new laptop and all my R libraries are gone, I got errors for missing package DT and pander (couldn't load just with `library()`, had to manually install them). I think we had this issue before, so maybe add them as part of the generic `install.packages()` call (in compile_tables?) - it would only be an issue for someone who hasn't run any automations before.

- Reading input data from example folder. Having sorted out the packages, I'm getting an error:
`Error in file(file, "rt") : cannot open the connection`
It's because the variable in example_config is named "Example_Input", but the folder is named "Example Input", so either one needs to be edited - otherwise the test data cannot be found. It's an easy fix and the main script runs after that.
- The produced output file (Example_Output, in this case) the Region and Local authority disaggregations have NA values. In addition, Ethnic group disaggregation is only blank, so not sure if this is expected (the live version has values for it, but it may be that this was cut out for the test data?). There was no existing test output to check against, so it may be that these values were expected, though not sure about the NAs.
- Error in the checks Rmd file:
`Quitting from lines 36-81 (4-1-1_checks.Rmd)` Looking into it in more detail, the problem is actually at line 61: `new_data <- read.csv(paste0("Output/", date, "_4-1-1", ".csv"))` 
It's looking for the comparison data in Output folder, rather than Example_Output (since I'm running the test version). So basically the name of the folder is hard-coded instead of using the variable assigned in the config script (easy fix too!). Should be `new_data <- read.csv(paste0(output_folder,"/", date, "_4-1-1", ".csv"))`
- fixing the above leads to another error in the checks file:
`Quitting from lines 36-81 (4-1-1_checks.Rmd)` 
`Error in rename():`
`! Can't rename columns that don't exist.`
`x Column Local.Authority doesn't exist.`
It's because in the new_data (line 63) the variable is actually `Local.authority` (not capitalised), so it doesn't find the column for renaming.
- Fixing the column name leads to another error `Quitting from lines 88-107 (4-1-1_checks.Rmd) `, specifically it some from line 94, because the `comparison` table doesn't have a column called `new`. So something goes wrong at the creation of `comparison` from `joined_data`, but not sure what. The dataset and values columns seem alright, but it's not picking up the 'new' marker from dataset. May be to do with the NAs and empty cols in the csv output, so can't match/pivot old and new?

Basically, can you do a test run on your side and see if you are also getting these errors?

**Comment by MichaelGNairn** [comment]
Thanks for this Atanaska, all done now. 

FInal point appears to only be an issue for the test run with example data, and not for the real run with the full datasets. 

I'll try and remedy later this morning.

**Review by ANikolova22 — APPROVED** [review]
Ok, looks good! I think the most important bit was to add the packages and re-coding the output variable
